examples/CMakeLists.txt

```diff
@@ -23,17 +23,10 @@ blt_add_executable(
   SOURCES chai-umpire-allocators.cpp
   DEPENDS_ON ${chai_umpire_example_depends})
 
-blt_add_test(
```

Comment:
Hmm, deleting tests?

Reply:
this target didn't exist when I tried to rebuild - maybe my workspace was hosed? Also it looks like this test exists in the integration tests, not here in examples...

Reply:
Well, fixed my build issue. But this is still a duplicate entry - the target is also is added in tests/integration and in fact it is being tested twice on develop.